src/locales/pl-pl.js

```diff
@@ -0,0 +1,107 @@
+export const isSingular = (count) => parseInt(count, 10) === 1;
```

Comment:
I think this needs to be the same as the `isSingular` function in `en-gb.js`, only passing `count` into `parseInt` if it's a string

Reply:
For more context `count` can be a string or a number, and `parseInt` can only accept a string as it's first parameter, so trying to use this in TypeScript is actually invalid:

![image](https://user-images.githubusercontent.com/14963680/134896704-c152346f-7055-42cc-9dc0-f3b4034be81c.png)